tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -82,18 +82,131 @@ int main() {
     vformat_to(back_insert_iterator(output_string), locale::classic(), "{0} {0}", make_format_args((const char*) "f"));
     assert(output_string == "f f");
 
-    // TODO: enable these in _Write function PR for sv and bool
-    /*
+    // Test string_view
     output_string.clear();
     vformat_to(back_insert_iterator(output_string), locale::classic(), "{}", make_format_args("f"sv));
     assert(output_string == "f");
-    */
 
-    /*
+    // Test bool
     output_string.clear();
     vformat_to(back_insert_iterator(output_string), locale::classic(), "{}", make_format_args(true));
     assert(output_string == "true");
+
+    output_string.clear();
+    vformat_to(back_insert_iterator(output_string), locale::classic(), "{}", make_format_args(false));
+    assert(output_string == "false");
+
+    // Test char
+    output_string.clear();
+    vformat_to(back_insert_iterator(output_string), locale::classic(), "{}", make_format_args('a'));
+    assert(output_string == "a");
+
+    // Test void*
+    output_string.clear();
+    vformat_to(back_insert_iterator(output_string), locale::classic(), "{}", make_format_args((const void*) 0));

```

Comment:
We conventionally avoid C-style casts and `0`-as-null; this should be `static_cast<const void*>(nullptr)`.